hie.yaml

```diff
@@ -0,0 +1,7 @@
+cradle:
```

Comment:
If we are all using `stack` then we should have a `stack` cradle.

Reply:
There are some serious issues with HLS and the stack cradle right now. see [this comment](https://github.com/haskell/haskell-language-server/issues/735#issuecomment-1268839545) and [this issue](https://github.com/haskell/haskell-language-server/issues/366)

Reply:
FWIW I have a similar experience with the `cabal` cradle.

Reply:
As in, you are having the problems in those issues with the current cradle?

Reply:
No, I have not run into any trouble recently in this package with either the `stack` or `cabal` cradle.  (🤞🏻) But at work I run into the issue under `stack` and for other home projects I ran into it with `cabal` (I don't recall when exactly tho).

Reply:
This is not a blocker for me.  I'm perfectly fine with using the `cabal` cradle for now.  The main pain point is that if you change `package.yaml`, then you need to invoke `stack` (or `hpack`) directly and update the `cabal` file so that HLS can pick up the changes.

Reply:
Got it. That's enough of a reason for me to change it back. Hopefully I don't run into it again.